StatsDownload/StatsDownload.TestHarness/MainForm.cs

```diff
@@ -41,13 +39,15 @@ private void CreateFileDownloadServiceAndPerformAction(Action<IFileDownloadServi
             var fileDownloadService = WindsorContainer.Instance.Resolve<IFileDownloadService>();
             fileDownloadServiceAction?.Invoke(fileDownloadService);
             CreateSeparationInLog();
+            WindsorContainer.Instance.Release(fileDownloadService);
```

Comment:
Should this be in a try..finally, or is it not because this is just a test harness?

Reply:
yes it should be but the service really shouldn't ever allow an exception out so good practice would be to put it in a try..finally